usaspending_api/broker/management/commands/fpds_nightly_loader.py

```diff
@@ -197,10 +201,13 @@ def insert_new_fpds(to_insert, total_rows):
                 row['funding_agency_code'] = funding_cgac_code
 
             # Find the award that this award transaction belongs to. If it doesn't exist, create it.
-            awarding_agency = Agency.get_by_toptier_subtier(
+            awarding_agency = (Agency.get_by_toptier_subtier(
```

Comment:
Instead of hitting the database for every award, is it possible to use the subtier/agency mappings at the beginning of the file. If it would help improve performance.

Reply:
That's a good idea, but because it's a performance improvement rather than a functionality change, can we write an issue for it to make a separate PR?  I want to limit this PR's scope.

Reply:
We can add it to backlog.

Reply:
Created https://federal-spending-transparency.atlassian.net/browse/DS-1952